Mathlib/Data/Quot.lean

```diff
@@ -319,15 +319,32 @@ theorem Quotient.liftOn₂_mk {α : Sort*} {β : Sort*} {_ : Setoid α} (f : α
   rfl
 
 /-- `Quot.mk r` is a surjective function. -/
+theorem Quot.surjective_mk {r : α → α → Prop} : Function.Surjective (Quot.mk r) :=
+  Quot.exists_rep
+
+/-- `Quotient.mk` is a surjective function. -/
+theorem Quotient.surjective_mk {s : Setoid α} :
+    Function.Surjective (Quotient.mk s) :=
+  Quot.exists_rep
+
+/-- `Quotient.mk'` is a surjective function. -/
+theorem Quotient.surjective_mk' [s : Setoid α] :
+    Function.Surjective (Quotient.mk' : α → Quotient s) :=
+  Quot.exists_rep
+
+/-- `Quot.mk r` is a surjective function. -/
+@[deprecated Quot.surjective_mk (since := "2024-09-02")]
 theorem surjective_quot_mk (r : α → α → Prop) : Function.Surjective (Quot.mk r) :=
```

Comment:
Arguably you should use `deprecated alias` for these, which also adjusts the docstring. Maybe it's not worth the risk of changing the statement by accident though.